Ontology/Willow/Capability/Sensor/QuantitySensor/EnergySensor/Thermal/Billed/BilledNaturalGasThermalEnergy.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@id": "dtmi:com:willowinc:BilledNaturalGasThermalEnergy;1",
```

Comment:
In looking, it seemed that Natural Gas gets billed either in volume or mass. Do we need this Thermal Energy variant?